docs/workers.md

```diff
@@ -589,15 +589,15 @@ regular expressions:
 
     ^/_matrix/client/(r0|v3|unstable)/keys/upload
 
-If `use_presence` is False in the homeserver config, it can also handle REST
-endpoints matching the following regular expressions:
+If `use_presence` is set to `false` in the homeserver config, the frontend proxy can also handle REST
+endpoints matching the following regular expression:
 
     ^/_matrix/client/(api/v1|r0|v3|unstable)/presence/[^/]+/status
 
 This "stub" presence handler will pass through `GET` request but make the
 `PUT` effectively a no-op.
```

Comment:
It really doesn't matter whether `use_presence` is set to `false` or not:  any worker can now handle `^/_matrix/client/.../presence/[^/]+/status`. It is no longer a "stub".